web/src/components/map/box.js

```diff
@@ -55,12 +53,33 @@ export const MapBox = ({ isLoading, pollutants }) => {
           showCompass={false}
           className={"navigation-control"}
         />
-        {markers}
+        <SessionData pollutants={pollutants} />
       </ReactMapGL>
     </Container>
   );
 };
 
+/**
+ * Session data to be displayed on the map
+ * @property {Array<Pollutant>} pollutants
+ */
+const SessionData = ({ pollutants }) => {
```

Comment:
Now that the markers have been promoted to have their own component, could we spin the component out into its own file? perhaps `./map/session.js`? 

Also, making it a separate component adds another layer of inheritance. Adding a PropTypes for `SessionData` will help enforce the correct data.